keyboards/coseyfannitutti/discipline/discipline.h

```diff
@@ -48,6 +48,8 @@
 { K40, K41, K42, _x_, _x_, _x_, K46, _x_, _x_, K49, K4A, _x_, K4C, K4D, K4E} \
 }
 
+#define LAYOUT_65_ansi_blocker LAYOUT_65_ansi_2_right_mods
```

Comment:
IMO this should be inverted; `LAYOUT_65_ansi_blocker` should be the name of the macro, with `LAYOUT_65_ansi_2_right_mods` as the alias.

Other than that, looks good.